lib/index.js

```diff
@@ -71,5 +71,23 @@ Sentry.prototype.loaded = function() {
  */
 
 Sentry.prototype.identify = function(identify) {
-  window.Raven.setUser(identify.traits());
+  window.Raven.setUserContext(identify.traits());
 };
+
+/**
+ * Clean out null values
+ */
+
+function reject(obj) {
```

Comment:
I think i've seen a reject dependency in other integrations (but might have been server side).